src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -64,7 +64,8 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
   const shouldHideDetailsCreateAlertCTA =
     (is_in_auction && hasEnded) ||
     (is_in_auction && lotIsClosed(sale, saleArtwork)) ||
-    is_sold
+    is_sold ||
+    artwork.artists?.length === 0
```

Comment:
**suggestion(non-blocking):** what do you think of moving the `artwork.artists?.length === 0` in the beginning of the check (start of [line 65](https://github.com/artsy/force/pull/10351/files#diff-aacf74b81201e9b848f011f880a9891dc317f43d2a1c2d3098ad490d3eda6278R65)) in order to hide it before checking the rest?

Reply:
Yes, I agree. I have nothing against it 😅